torch/_refs/__init__.py

```diff
@@ -1762,6 +1768,75 @@ def narrow(a: TensorLikeType, dim: int, start: int, length: int) -> TensorLikeTy
     return prims.slice_in_dim(a, start, start + length, axis=dim)
 
 
+def _normalize(a, norm_dims, eps):
+    computation_dtype = utils.get_computation_dtype(a.dtype)
+    a_acc = prims.convert_element_type(a, computation_dtype)
+    biased_var, mean = var_mean(a_acc, dim=norm_dims, unbiased=False, keepdim=True)
```

Comment:
currently nvfuser doesn't understand var_mean, so this decomp is not readily usable

Reply:
Same here.

Also, the reason the previous decomp used separate calls to `var` and `mean` was that it more closely matched the numerics of eager mode. @ngimel's opinion was that the discrepancy shouldn't matter, but just noting in case you need to adjust tolerances on any tests.

Reply:
I will make `mean` working with the nvFuser executor in a separate PR. `var` will be implemented later as it's currently a "prim" in PyTorch, but a composite in nvFuser.

It's separate calls to `var` and `mean`. Take a look at the `var_mean` function:
```
def var_mean(
    a: TensorLikeType,
    dim: Union[Optional[int], Optional[List[int]]] = None,
    unbiased: Optional[bool] = None,
    keepdim: bool = False,
    *,
    correction: Optional[int] = None,
):
    v = var(a, dim, unbiased, keepdim, correction=correction)
    m = mean(a, dim, keepdim)
    return v, m
```

Reply:
PR for mean: https://github.com/pytorch/pytorch/pull/79444
PR for var: https://github.com/pytorch/pytorch/pull/79517
And together they make nvfuser understand var_mean.

Reply:
I believe @peterbell10 once told me that `var_mean` had some speed and stability problems in PyTorch (?). I do not remember very well so I'll let her discuss this, but perhaps this is relevant here.

Reply:
On CPU, the mean from `var_mean` is less accurate than calling `mean` separately. `mean` is implemented roughly as `sum(x, dim) / x.size(dim)`, where `sum` uses a low-error summation algorithm. `var_mean` on the other hand computes both the mean and var in a single pass, but with a naive summation that is less accurate.

Reply:
For similar reasons, can we change this call to a `torch.var_mean` as well?

Reply:
Well, then, unless you decompose `torch.var_mean` further, you'd get inaccurate cpu result